pyproject.toml

```diff
@@ -6,3 +6,9 @@ requires = [
     "setuptools_scm >= 2.0.0, <3"
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+DJANGO_SETTINGS_MODULE = "electionleaflets.settings.testing"
+python_files = ["test_*.py", "*_test.py"]
+addopts = "--reuse-db --tb=short -p no:warnings"
+django_debug_mode = true
```

Comment:
This does fix the tests locally for me :+1: 
I'm slightly unclear how they worked in CI or for you though. Can you give me a quick explanation of that?

Reply:
Yes.

`pytest-django`'s `live_server` runs the server with `DEBUG=False`. This, mixed with a combination of `django-pipeline` and `whitenoise` means that assets are not collected in to `static`. It's assumed that you've run `collectstatic` beforehand.

In CI I'm running `collectstatic`, and you could have got your tests working locally by doing this, however I think it's better if this isn't a required step to get tests running. 

The change I made gets `pytest-django` to turn _on_ `DEBUG` in `live_server`, meaning `whitenoise` will collect static files into `static` as part of the request that Playwright is making.

Reply:
Hmm. This is why I'm confused. As I said in the original post

> I tried running `./manage.py collectstatic` to see if that fixed it, but same error

but lets not get too bogged down in it

Reply:
Ah right, this is per our conversation this morning: `collectstatic --settings electionleaflets.settings.testing` isn't the same as not specifying the test settings.

Reply:
aah that was probably it